application/src/test/java/org/opentripplanner/routing/algorithm/filterchain/ItineraryListFilterChainTest.java

```diff
@@ -184,6 +184,7 @@ void testRoutingErrorsOriginDestinationTooCloseTest() {
       .withRemoveTransitWithHigherCostThanBestOnStreetOnly(
         CostLinearFunction.of(Duration.ofSeconds(0), 1.0)
       )
+      .withPageCursorInputSubscriber(i -> {})
```

Comment:
Since the builder provides the default, I guess these are not necessary?

Reply:
I removed these